source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -155,15 +168,20 @@ SimpleHttpCache::varyLookup(const LookupRequest& request,
     return SimpleHttpCache::Entry{};
   }
   ASSERT(iter->second.response_headers_);
+  Http::ResponseTrailerMapPtr trailers_map = nullptr;
```

Comment:
you don't need to init this; unique_ptr has a constructor which has the same effect.